### ISSUE TITLE
Slow Queries not showing

### ISSUE THREAD

**Issue by jubinmytheen** [ankane/pghero#462]
Hi,

I have installed pgHero tool on my server through Docker. And configured multiple databases on the dashboard. Unfortunately, I am not getting slow query details other than one database. The slow query details are showing only on one database. Already configured **pg_stat_statements** extension on the database. 

![slow_query_details](https://github.com/ankane/pghero/assets/112147946/b087c1d7-6417-4d3b-8507-84a384a9f61c)
![slow_query_not_shown](https://github.com/ankane/pghero/assets/112147946/eb2d2049-6d48-48ee-af37-005f49e67e9d)

I have only passed database connection string details through the pghero.yml file. Is there anything need to configure on the config file to show the slow query details?

Thanks!

**Comment by ankane** [comment]
Hi @jubinmytheen, all checks besides connections (which is instance-wide) will be for the selected database (there's not a global dashboard).

**Comment by jubinmytheen** [comment]
@ankane Thanks for your time. I understood that there is no global dashboard for this. We need to select the database to see the details for that database. In our context, we have added multiple databases to the pgHero, but only one database is returning the slow query details. Other databases are not showing slow query details.
We have already enabled the slow queries on database server. Is there anything specifically to do for showing slow queries?
